indra/newview/lluilistener.cpp

```diff
@@ -97,6 +99,41 @@ LLUIListener::LLUIListener():
         "Add menu separator to the [\"parent_menu\"] within the Top menu.",
         &LLUIListener::addMenuSeparator,
         llsd::map("parent_menu", LLSD(), "reply", LLSD()));
+
+    add("setMenuVisible",
+        "Set menu [\"name\"] visibility to [\"visible\"]",
```

Comment:
Question: I called the top-menu query method `getParents()` because the only existing use for those names was to pass `parent_menu` parameters to other methods. This method makes me wonder whether I should instead have called it `getMenus()` or `getTopMenus()`?

Reply:
I think I like `getTopMenus()` more. `getParents()` is quite general name.